addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -109,6 +109,12 @@ export class LinkPopover extends Component {
             this.props.onClose();
         }
     }
+    onKeydownEnter(ev) {
+        if (ev.key === "Enter") {
+            ev.preventDefault();
```

Comment:
:magic_wand:

Reply:
https://github.com/odoo/owl/blob/master/doc/reference/event_handling.md#modifiers

Reply:
oops, the `.prevent` modifier would actually prevent default on any key down, which is not what we want.
It's good the way it is!
Just add a commit message and it's good to go.